### ISSUE TITLE
Clojurescript is a hard dependency

### ISSUE THREAD

**Issue by codahale** [metametadata/clj-fakes#1]
I’m using clj-fakes in a regular Clojure application and it’s great. That said, I haven’t been able to exclude the Clojurescript dependency:

```
Exception in thread "main" java.io.FileNotFoundException: Could not locate cljs/analyzer__init.class or cljs/analyzer.clj on classpath., compiling:(clj_fakes/reflection.clj:1:1)
```

Would it be possible to change `clj-fakes.reflection` to use reader conditionals? I’m happy to help out with a pull request, but I have roughly no experience with Clojurescript.

**Comment by metametadata** [comment]
Thank you for the feedback!

I'll try to make ClojureScript a provided dependency for the lib, so that you won't have to manually exclude it in Clojure-only project.

**Comment by metametadata** [comment]
Fixed in 0.4.0. ClojureScript should now be provided by library consumer.

**Comment by codahale** [comment]
Awesome! Thank you so much.
